guide/14-deep-learning/inference-only-text-models-in-arcgis-learn.ipynb

```diff
@@ -0,0 +1,987 @@
+{
```

Comment:
<p><s>These</s> <s>models</s> <s>have</s> The model has been fine-tuned on a question answering...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/851/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>Made the necessary changes</p>